arm/Microsoft.Sql/servers/deploy.bicep

```
@@ -1,9 +1,9 @@
-@description('Required. Administrator username for the server.')
-param administratorLogin string
+@description('Conditional. Administrator username for the server.')
+param administratorLogin string = ''
 
-@description('Required. The administrator login password.')
+@description('Conditional. The administrator login password.')
```

We currently don't support 'Conditional' as a keyword (Pester will aks for `Required`, `Optional` or `Genererated`). However, might be worth adding in a subsequent PR.

```suggestion
@description('Optional. The administrator login password.')
```